code_schemes/rqas/aik/aik_household_income.json

```diff
@@ -13,6 +13,78 @@
             "VisibleInCoda": true,
             "Shortcut": "e"
         },
+        {
+            "CodeID": "code-4599d6d5",
+            "CodeType": "Normal",
+            "DisplayText": "under ksh 10,000",
+            "NumericValue": 1,
+            "StringValue": "under_ksh_10000",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-dd3195cb",
+            "CodeType": "Normal",
+            "DisplayText": "ksh 10,001 - 20,000",
+            "NumericValue": 2,
+            "StringValue": "ksh_10001_20000",
```

Comment:
Would `ksh_10001_to_20000` be easier to read in analysis?

Reply:
Updated thanks.